lexxpluss_apps/src/board_controller.cpp

```diff
@@ -466,32 +471,46 @@ class emergency_switch_set { // Variables Implemented
 
 class wheel_switch { // Variables Implemented
 public:
-    void set_disable(bool disable) {
-        if (disable) {
-            gpio_dt_spec gpio_dev = GET_GPIO(wheel_en);
-            if (!gpio_is_ready_dt(&gpio_dev)) {
-                LOG_ERR("gpio_is_ready_dt Failed\n");
-                return;
+    void poll() {
+        // update  
+        if (pending_left_right_disable.has_value()) {
```

Comment:
memo: pending_left_right_disable is disabled during soft braking. Otherwise, it is std::nullopt.
The value is only set during soft braking.